### PR TITLE
Issue#11-Jatin-Add URL navigation functionality to Source Code ListTile

### PR DESCRIPTION
**Pull request by JatinSharma32**
Contribution Guidelines
 - [x] Ensure that your contribution is relevant to the goals and scope of the project.
 - [x] Make sure your changes do not introduce new bugs and pass all existing tests.
 - [x] Keep your code clean, well-organized, and documented.
 - [x] Provide clear and comprehensive commit messages.
 - [ ] Include relevant documentation updates if applicable.
 - [x] Be respectful and considerate in your interactions with other contributors.

This PR is for [Issue#11](https://github.com/Spyy004/LifeList/issues/11)

### CODE REVIEW

File: lib/screens/settingsscreen.dart [hunk 1]
```diff
@@ -65,6 +66,15 @@ class SettingsScreen extends StatelessWidget {
                   text: SOURCE_CODE,
                   style: Theme.of(context).textTheme.displaySmall,
                 ),
+                onTap: () async {
+                 const url = 'https://github.com/Spyy004/LifeList';
```
Comment (Spyy004) [inline lib/screens/settingsscreen.dart:70]: use a const to store the uri and store it in consts.dart

Comment (Spyy004) [inline lib/screens/settingsscreen.dart:70]: 1) Everything that is present in onTap is correct but it should be put inside a function which is present outside the build method. Call that function from here.
2) The uri must be stored in a const and must be used from that const variable. Hardcoding it is not a good practice.

**Review by Spyy004 — COMMENTED** [review]
Some changes need to made in settingsscreen.dart. Please have a look and do the same. Also, I have merged the themes.dart file in the code. 
So take a git pull and push the code again. The themes.dart file will be same for both of us and it won't appear in this PR.

**Comment by Spyy004** [comment]
any updates?

**Comment by JatinSharma32** [comment]
## Closing this
For less complexity, I have made a new [PR](https://github.com/Spyy004/LifeList/pull/23)

**Pull request closed without merging**